### DISCUSSION TITLE
Where are stored the contents (the menu items) ?

### DISCUSSION THREAD

**Discussion by areavailable** [Open-Shell/Open-Shell-Menu#1946] [Q&A]
The menu items and subitems

**Comment by ge0rdi** [comment]
Right-click on Start button and select `Open` (or `Open All Users`).

**Reply by areavailable** [reply]
Thank you for replying.

![whereareitems](https://github.com/user-attachments/assets/06ecee2c-7ec6-40e8-b2ee-a0e439a4b9fe)

**Reply by ge0rdi** [reply]
Those look like _recently/frequently used_ programs (though hard to say as the menu on the picture is not complete and not sure what your settings are). They are populated automatically based on what you are using the most.

You can change settings for them in `Show recent or frequent programs` setting.

**Reply by areavailable** [reply]
Not  the recently/frequently used.  A static list.
Menu's left column (of two) is in picture.

**Reply by ge0rdi** [reply]
Oki, you mean _pinned_ items. Those are stored in `%APPDATA%\OpenShell\Pinned`.

You can right-click on an item there, select `Properties` and in `General` tab you will see path to the shortcut file.
